script.js

```diff
@@ -0,0 +1,63 @@
+//Greet player
+var name = prompt("What is your name?");
+alert("Hello " + name +".");
+
+var event = prompt("You've been invited to three events: a party, a happy hour and a coding meetup. Which do you choose? (P, H, M)");
+
+if(event === "P"){
+  alert("Parties are awesome!")
+
+  var someoneKnown = prompt("You arrive at the party and see someone you know. Is it your Ex, an old Roommate, or a Boss? (E, R, F)")
+      if (someoneKnown === "E"){
```

Comment:
no need to indent after a variable or for an if, above you had already indented the alert because were inside an if condition.